server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -719,6 +719,11 @@ public class ShuffleServerConf extends RssBaseConf {
           .booleanType()
           .defaultValue(false)
           .withDescription("Whether to enable app detail log");
+  public static final ConfigOption<Boolean> SERVER_TRIGGER_REPORT_WHILE_UNREGISTER_ENABLED =
```

Comment:
How about making this option rename to `rss.server.heartbeatReportOnUnregisterEnabled` ?

Reply:
done